solidity-sdk/README.md

```diff
@@ -0,0 +1,66 @@
+## Foundry
```

Comment:
might be good to update the readme at this point too, I guess this is just the default?